cmake/Modules/LocalLibsharpyuv.cmake

```diff
@@ -1,10 +1,62 @@
+set(AVIF_LOCAL_LIBSHARPYUV_GIT_TAG v1.3.2)
+
 set(LIB_FILENAME "${CMAKE_CURRENT_SOURCE_DIR}/ext/libwebp/build/libsharpyuv${CMAKE_STATIC_LIBRARY_SUFFIX}")
-if(NOT EXISTS "${LIB_FILENAME}")
-    message(FATAL_ERROR "libavif(AVIF_LIBSHARPYUV=LOCAL): ${LIB_FILENAME} is missing, bailing out")
-endif()
+if(EXISTS "${LIB_FILENAME}")
```

Comment:
All those checks for a static lib are for backward compatibility right? In case somebody has built those libs locally. We could decide that this is not supported which would result in less code right ?